src/components/cookies/CookieConsent.tsx

```diff
@@ -4,12 +4,11 @@ import CookieConsentBanner from 'react-cookie-consent';
 import { Link } from 'src/components/primitives/Link';
 import { useRootStore } from 'src/store/root';
 
-import CookieConfigModal from './CookieConfigModal';
-
 export default function CookieBanner() {
```

Comment:
I think to save some future confusion we should rename any components or state variables from Cookie -> Analytics.

Reply:
done here `bbdab4e39cfe9181c05300f77fc342581dca429f`